src/server/config.js

```diff
@@ -16,6 +16,7 @@ var simulationFilePath;
     { name: 'projectRoot', single: true },
     { name: 'server', optional: true },
     { name: 'simHostOptions' },
+    { name: 'simulationFilePath' },
```

Comment:
Oh I see that this is an internal property which is configured based on the user-provided `simulationpath`, so it's fine.